ui/apps/platform/src/Containers/NetworkGraph/components/EmptyUnscopedState.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Bullseye, Title } from '@patternfly/react-core';
+import { ModuleIcon } from '@patternfly/react-icons';
+import { SVGIconProps } from '@patternfly/react-icons/dist/esm/createIcon';
+
+import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate';
+
+function EmptyIcon(props: SVGIconProps) {
+    return (
+        <ModuleIcon {...props} size="lg" style={{ color: 'var(--pf-global--palette--red-100)' }} />
```

Comment:
I know the mock has a red color, but now that I look at it, it feels kinda strong. I don't know if anyone else feels that way too